config/locales/views/admin/en.log_events.yml

```diff
@@ -13,7 +13,10 @@ en:
         journal:
           create: Journal created
         statement:
-          create: Statement created
+          create: Statement added
```

Comment:
Why did you change this? I don't have a strong preference for "created" vs "added", but I think we should be consistent with journals.

Reply:
Sorry I'll change this back, probably I miss it when I merge master and fixed conflicts.